Assuming unrestricted shared filesystem usage.
None
host: incline31
Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 2
Rules claiming more threads will be scaled down.
Job stats:
job                                          count
-----------------------------------------  -------
all                                              1
extract_variants_tidy_gnomad_variants            1
tidy_data_for_plots_parse_gnomad_variants        1
total                                            3

Select jobs to execute...
Execute 1 jobs...
[Thu Sep 18 11:19:59 2025]
localrule extract_variants_tidy_gnomad_variants:
    input: data/interim/gnomad_snrna_variants.vcf.gz, data/interim/snrnas.bed, experiments/extract_variants/filter_gnomad_variants.sh
    output: data/interim/gnomad_snrna_variants_tidy.tsv, data/interim/header_bed_info.txt
    jobid: 3
    reason: Missing output files: data/interim/gnomad_snrna_variants_tidy.tsv; Code has changed since last execution
    resources: tmpdir=/tmp
Activating conda environment: vep
[Thu Sep 18 11:20:16 2025]
Finished jobid: 3 (Rule: extract_variants_tidy_gnomad_variants)
1 of 3 steps (33%) done
Select jobs to execute...
Execute 1 jobs...
[Thu Sep 18 11:20:16 2025]
localrule tidy_data_for_plots_parse_gnomad_variants:
    input: data/interim/gnomad_snrna_variants_tidy.tsv, experiments/tidy_data_for_plots/parse_gnomad_variants.py
    output: data/interim/gnomad_snrna_variants_tidier.tsv
    jobid: 5
    reason: Missing output files: data/interim/gnomad_snrna_variants_tidier.tsv; Input files updated by another job: data/interim/gnomad_snrna_variants_tidy.tsv
    resources: tmpdir=/tmp
Activating conda environment: crdg_test
RuleException:
CalledProcessError in file "/mnt/bmh01-rds/Ellingford_gene/crdg_interview/workflow/tidy_data_for_plots.smk", line 14:
Command 'source /mnt/iusers01/bk01/m40482ab/miniforge3/bin/activate 'crdg_test'; set -euo pipefail;  python 3 -m experiments/tidy_data_for_plots/parse_gnomad_variants.py' returned non-zero exit status 2.
[Thu Sep 18 11:20:16 2025]
Error in rule tidy_data_for_plots_parse_gnomad_variants:
    message: None
    jobid: 5
    input: data/interim/gnomad_snrna_variants_tidy.tsv, experiments/tidy_data_for_plots/parse_gnomad_variants.py
    output: data/interim/gnomad_snrna_variants_tidier.tsv
    conda-env: crdg_test
    shell:
        python 3 -m experiments/tidy_data_for_plots/parse_gnomad_variants.py
        (command exited with non-zero exit code)
Shutting down, this might take some time.
Exiting because a job execution failed. Look below for error messages
[Thu Sep 18 11:20:21 2025]
Error in rule tidy_data_for_plots_parse_gnomad_variants:
    message: None
    jobid: 5
    input: data/interim/gnomad_snrna_variants_tidy.tsv, experiments/tidy_data_for_plots/parse_gnomad_variants.py
    output: data/interim/gnomad_snrna_variants_tidier.tsv
    conda-env: crdg_test
    shell:
        python 3 -m experiments/tidy_data_for_plots/parse_gnomad_variants.py
        (command exited with non-zero exit code)
Complete log(s): /mnt/bmh01-rds/Ellingford_gene/crdg_interview/.snakemake/log/2025-09-18T111958.960075.snakemake.log
WorkflowError:
At least one job did not complete successfully.
